src/dmd/backend/global.d

```diff
@@ -140,7 +140,8 @@ void util_set32();
 void util_set64();
 int ispow2(uint64_t);
 
-//#if __GNUC__
+version (Posix)
+{
```

Comment:
shouldn't the util_malloc/calloc/free functions use (templated) forwarding functions to not repeat the `version(Posix)` condition everywhere? Or just implement `util_*` in util2.c for Posix, too? The parc_* don't seem to be used anyway.

Reply:
The memory allocation functions in the back end are a bit of a mess. Straightening them out is something I plan to do, but it's beyond the scope of this PR.